omero/users/cli/import.txt

```diff
@@ -15,91 +15,136 @@ Import command
+.. option:: --java-help
 
-The :option:`--depth` option sets the number of directories to scan down for
-files (default: 4)::
+    Display the help for the Java option of the import command
 
-    $ bin/omero import --depth 1 images_folder
+    Java options can be passed after `--`, e.g.::
 
-Java Command Line Importer
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+       bin/omero import image.tif -- --name=test --description=description
 
-There are three ways to use the importer from the command line:
+.. option:: --advanced-help
 
-* ``importer-cli`` shell script, included for Linux and Macintosh only
-* ``ome.formats.importer.cli.CommandLineImporter`` Java class
-* ``bin/omero import`` command, from a server installation only
+    Display the advanced help for the import command
 
-Using the importer using the shell script might look like this:
+    Some advanced import options are described in the :ref:`inplace_import`
+    section.
 
-::
+.. option:: -f
 
-    ./importer-cli -s localhost -u user -w pass path/to/file
+    Display all the used files for importing then exits::
 
-To use the ``ome.formats.importer.cli.CommandLineImporter`` class from java on
-the command line you will also need to include a classpath to the required
-support jars. Please look inside of the ``importer-cli`` script for an example
-of how to do this.
+        $ bin/omero import -f image.tif
+        $ bin/omero import -f images_folder
 
-With a server installation, from server directory, execute the following
-command:
+    This will output a list of all the files which would be imported in groups
+    separated by "#" comments. Note that this usage does not require a running
+    server to be available.
 
-::
+.. option:: --depth DEPTH
 
-    bin/omero -s localhost -u user import path/to/file
+    Set the number of directories to scan down for files (default: 4)::
 
-Note that this method will prompt for any mandatory arguments not provided.
+        $ bin/omero import --depth 1 images_folder
 
-The importer can also be used to detect what files *would* be imported:
+.. option:: --skip SKIP
 
-::
+    Specify optional step to skip during import.
 
-    ./importer-cli -f /path/to/file
+    The import of very large datasets like High-Content Screening data or
+    SPIM data can be time and resource consuming both at the client and at the
+    server level. This option allows the disabling of some non-critical steps
+    and thus faster import of such datasets. The caveat associated with its
+    usage is that some elements are no longer generated at import time and may
+    either be re-generated during client access (e.g. thumbnails) or may need
+    manual intervention to be generated (e.g. minmax).
+    Available options that can be skipped are currently:
 
-or
+    all
+        Skip all optional steps described below
+    checksum
+        Skip checksum calculation on files after transfer
+    minmax
+        Skip computation of the minima and maxima pixel values
```

Comment:
Should it be mentioned here of in the paragraph above that this option includes skipping the checksum...